core/js/src/main/scala/com/softwaremill/sttp/FetchBackend.scala

```diff
@@ -0,0 +1,46 @@
+package com.softwaremill.sttp
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+import scala.scalajs.js
+import scala.scalajs.js.JSConverters._
+import scala.scalajs.js.Promise
+import scala.scalajs.js.annotation.JSExport
+
+import org.scalajs.dom.experimental.BodyInit
+import org.scalajs.dom.experimental.{Response => FetchResponse}
+
+class FetchBackend private (fetchOptions: FetchOptions)(implicit ec: ExecutionContext)
+    extends AbstractFetchBackend[Future, Nothing](fetchOptions)(new FutureMonad()) {
+
+  @JSExport("send")
+  def send_JS[T](request: Request[T, Nothing]): Promise[Response[T]] = send(request).toJSPromise
```

Comment:
out of curiosity, why does it need to have a different name?

Reply:
hm in fact I don't understand this overall construction - there's a `send` in `AbstractFetchBackend`, why do you need another send over here? (which in JS shadows the one from the abstract class)

Sorry if this is something basic from scala.js :)

Reply:
It exposes that a method named `send` to JavaScript code (normal JavaScript code cannot handle Scala.js code). As I am not exporting a way to create the class from JavaScript or create a request it is pointless so I will remove it.